lib/dolly/query.rb

```diff
@@ -18,7 +18,10 @@ def find *keys
         if keys.count > 1
           build_collection( query_hash )
         else
-          self.new.from_json( database.all_docs(query_hash).parsed_response )
+          doc = self.new
+          response = database.all_docs(query_hash).parsed_response
```

Comment:
right, as I said we now expect a parsed Hash and the mocks are returning a json string...